### PR TITLE
Update project setup for distribution

### PR DESCRIPTION
**Pull request by maxwalls**

### CODE REVIEW

File: .fmt/install-githook.sh [hunk 1]
```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+set -e
+
+cd "$(dirname "$0")"
+fmt_dir="$PWD"
+cd ..
+cwd="$PWD"
+hooks_dir="$cwd/.git/hooks"
+fmt_venv="$fmt_dir/venv"
```
Comment (mikaelforeseeti) [inline .fmt/install-githook.sh]: call i .venv so it's a bit more discreet?

File: README.md [hunk 2]
```diff
@@ -57,9 +58,9 @@ If you wish to run securiCAD Vanguard with a local file, replace the `client.get
 ```python
 import json
 
-with open('data.json', 'r') as json_file:
+with open('data.json', encoding='utf-8') as json_file:
```
Comment (mikaelforeseeti) [inline README.md]: I assume 'r' is the default, but I'd like to still keep it for clarity

File: generate_requirements.sh [hunk 3]
```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
```
Comment (mikaelforeseeti) [inline generate_requirements.sh]: set -e ?

File: setup.py [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+    license="The Apache Software License, Version 2.0",
+    classifiers=[
+        "Intended Audience :: Developers",
+        "Intended Audience :: Information Technology",
+        "Natural Language :: English",
+        "License :: OSI Approved :: Apache Software License",
+        "Operating System :: OS Independent",
+        "Programming Language :: Python",
+        "Programming Language :: Python :: 3",
+        "Programming Language :: Python :: 3.6",
```
Comment (mikaelforeseeti) [inline setup.py]: Does 3.6 support format strings?

File: .fmt/install-githook.sh (hunk 1 above)
Comment (maxwalls) [reply .fmt/install-githook.sh]: It's already under `.fmt/venv`, which I think is discreet enough

File: setup.py (hunk 4 above)
Comment (maxwalls) [reply setup.py]: Introduced in 3.6

File: .fmt/install-githook.sh (hunk 1 above)
Comment (mikaelforeseeti) [reply .fmt/install-githook.sh]: Ah, missed that. Agreed.

**Pull request merged**